### ISSUE TITLE
DIR inaccessibility

### ISSUE THREAD

**Issue by KXzeno** [KXzeno/WorKx#3]
The _img_ function in 404.html cannot output images from ../static and ../assets, possible inaccessibility to the directories.
Cannot put image in the same DIR, it is meant for templates which will invoke server runtime error

**Comment by KXzeno** [comment]
Went to several documentations; hugo was most crucial. 

Strong discovery made using web console, identifying the sources that the 404 page used, '/img' is seen to have only one file which is affected by a background template located ./layouts/partials

Upon reviewing the template's code, it uses the function ```resources.Get()``` to obtain data at the root or outside current directory. With the guide of hugo documentation, I was able to use the command to service 404.html the same way; I used the get for the gif I wanted to display; succeeded.
